.github/workflows/main.yml

```diff
@@ -23,9 +23,15 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-
-      # Runs a set of commands using the runners shell
-      - name: Run and test
-        run: |
-          npm install
-          npm test
+      # Cache the npm install cache
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Is there documentation for where this incantation comes from?

Reply:
Yeah, copied directly from https://github.com/actions/cache/blob/main/examples.md#node---npm